src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/annotations/AExpect.java

```diff
@@ -53,7 +53,8 @@
    * Currently supported places to look for RCA outputs are the SQLite file or by hitting the rest endpoint.
    */
   enum Type {
-    REST_API
+    REST_API,
+    SQLITE
```

Comment:
we might want to say DB_QUERY.

I hope our persistence layer is pluggable enough that we can swap out sqlite for anything quite easily :)

Reply:
Right. Modified to DB_QUERY